src/Cake.Markdownlint/Cake.Markdownlint.csproj

```diff
@@ -23,8 +23,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Cake.Core" Version="0.28.0">
-      <PrivateAssets>all</PrivateAssets>
+    <PackageReference Include="Cake.Core" Version="0.33.0">
+      <PrivateAssets>All</PrivateAssets>
```

Comment:
I just noticed that AddinDiscoverer changed the casing of the `PrivateAssets` value. I don't know if the value is case-sensitive or not. If so, what is the proper casing? 

This [example](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets) seems to indicate that it should be all lower-case but the article does not specifically address case-sensitivity.

Reply:
In this other [article](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#packagereference), the first letter in `All` is capitalized.

Reply:
Investigating this situation on GitHub, I found a mix of "all" and "All": https://github.com/search?q=PrivateAssets+all&type=Code

This leads me to conclude that this value is not case sensitive and therefore your lower-case value should not have been changed by the AddinDiscoverer. I will fix this PR to restore your original value.